src/admin/class-options-inputs.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Duplicate Post plugin file.
+ *
+ * @package Yoast\WP\Duplicate_Post\Admin
+ */
+
+namespace Yoast\WP\Duplicate_Post\Admin;
+
+/**
+ * Class Options_Inputs
+ */
+class Options_Inputs {
+
+	/**
+	 * Creates a basic input based on the passed parameters.
+	 *
+	 * @param string $type          The type of input.
+	 * @param string $name          The name of the input.
+	 * @param string $value         The value of the input.
+	 * @param string $id            The ID of the input.
+	 * @param string $attributes    The additional attributes to use. Optional.
+	 *
+	 * @return string The input's HTML output.
+	 */
+	protected function input( $type, $name, $value, $id, $attributes = '' ) {
+		return sprintf(
+			'<input type="%s" name="%s" id="%s" value="%s" %s />',
+			\esc_attr( $type ),
+			\esc_attr( $name ),
+			\esc_attr( $id ),
+			\esc_attr( $value ),
+			$attributes
+		);
+	}
+
+	/**
+	 * Creates a checkbox input.
+	 *
+	 * @param string $name The name of the checkbox.
+	 * @param string $value The value of the checkbox.
+	 * @param string $id The ID of the checkbox.
+	 * @param bool   $checked Whether or not the checkbox should be checked.
+	 *
+	 * @return string The checkbox' HTML output.
+	 */
+	public function checkbox( $name, $value, $id, $checked = false ) {
+		$checked = $checked ? 'checked="checked"' : '';
```

Comment:
Do we need to escape the double quotes? Not sure, double checking 😄

Reply:
Double-qoutes within single quotes are, as far as I know, treated as if they're part of a string and nothing more.